.circleci/config.yml

```
@@ -128,3 +133,21 @@ jobs:
           name: Test
           command: |
             hack/test.sh
+
+workflows:
+    version: 2
+    helm-chart:
+        jobs:
+            - helm/lint-chart:
+                  filters:
+                      tags:
+                          ignore: /.*/
+
+            - helm/publish-chart:
+                  context: helm
+                  filters:
+                      branches:
+                          ignore: /.*/
+                      tags:
+                          only: /chart\/.*\/\d+.\d+.\d+.*/
+
```

`build` job must be included in workflow section

```suggestion
  build:
    jobs:
      - build
```